onnxruntime/core/providers/xnnpack/xnnpack_execution_provider.cc

```diff
@@ -142,6 +142,10 @@ static void AddComputeCapabilityForNodeUnit(const NodeUnit& node_unit,
     }
     sub_graph->SetMetaDef(FuseQDQGroup(node_unit));
     sub_graph->use_existing_schema = true;
+    // layout insensitive ops, we will create schema in CreateFusedSubGraphNode.
+    if (node_unit.OpType() == "Softmax") {
+      sub_graph->use_existing_schema = false;
+    }
   } else {
```

Comment:
If we let CreateFusedSubGraphNode do it, it create a new schema for every Softmax node. 

Can we instead create and register the schema in the EP and leave use_existing_schema as true, so that when we do the schema lookup a single shared 'xnnpack Softmax' schema is found in the xnnpack EP's registry?